pkg/scripts/tempesta_installer.sh

```diff
@@ -18,29 +18,34 @@
 # this program; if not, write to the Free Software Foundation, Inc., 59
```

Comment:
Please update the copyright year or (better) test the recent Anton's github hook that it does update the copyright

Reply:
> Please update the copyright year or (better) test the recent Anton's github hook that it does update the copyright

Done (by hook)